test/cql-pytest/nodetool.py

```diff
@@ -77,6 +77,12 @@ def flush(cql, table):
     else:
         run_nodetool(cql, "flush", ks, cf)
 
+def flush_keyspace(cql, ks):
```

Comment:
I think this is fine, and would keep it, but if you really wanted to allow the original flush() to work also for keyspace (like you said in the commit message), there is a fairly simple solution - you could simply check if table has no dot, and  if not, treat the parameter as a keyspace.
But I think what you did here is better.